src/test/java/AppHealthCheckTest.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AppHealthCheckTest {
+
+    @Test
+    public void isAppAlive() throws Exception {
+        HttpClient client = HttpClient.newHttpClient();
```

Comment:
Just a thought - instead of using the default Java 11 HTTP client, why not try out `@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)` and `TestRestTemplate`? It might get us closer to how the app behaves in the wild.

Also, since the port and base path could change down the line, what if we moved them to class fields? Might save us some headache later. Oh, and there's `@LocalServerPort` that's pretty neat for grabbing the port during tests. It's a shortcut for retrieving the port that is being used by an embedded Tomcat instance during tests.

And, just throwing it out there, maybe we can play around with a `@TestConfiguration` with mechanisms like `RestTemplateBuilder` for testing our auth flow? But that's just a suggestion. What do you think?